lib/version.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+define('CYPHT_VERSION', '2.5.2');
+
+$releases = json_decode(file_get_contents('https://github.com/cypht-org/cypht/blob/master/releases.json?raw=true'), true);
```

Comment:
Since you are fetching github already, why not using the data available there?
https://api.github.com/repos/cypht-org/cypht/releases